components/brave_wallet_ui/panel/container.tsx

```diff
@@ -381,6 +383,18 @@ function Container (props: Props) {
     // Logic here to cancel connecting your hardware wallet
   }
 
+  const removeSitePermission = (origin: string, address: string) => {
+    props.walletActions.removeSitePermission({ origin: origin, account: address })
+  }
+
+  const isConnectedToSite = React.useMemo((): boolean => {
+    if (activeOrigin === 'chrome://wallet/') {
+      return true
+    } else {
+      return connectedAccounts.includes(selectedAccount)
```

Comment:
Probably a gap in my JS knowledge, but is this operation safe? I don't know why it works here, but here's a small snippet to demonstrate my point.

```js
> [{foo: 'bar'}].includes({foo: 'bar'})
false
```

Reply:
Nice use of `.some`!